src/back/spv/layout.rs

```diff
@@ -24,6 +24,10 @@ impl PhysicalLayout {
         sink.extend(iter::once(self.bound));
         sink.extend(iter::once(self.instruction_schema));
     }
+
+    pub(super) fn supports_storage_buffers(&self) -> bool {
+        self.version >= 0x10300
+    }
```

Comment:
Its nice to already have this, but for now the back-end only supports SPIR-V 1.0. I don't mind including it already, but just to avoid confusing, there is a chance that SPIR-V version > 1.0 won't work.

Reply:
yeah, I understand. If we have a separate version check and error on anything other than 1.0, that's not going to make this code less needed. In the end, we'll want to support higher SPIR-V versions